server/lib/payments.js

```diff
@@ -590,6 +592,28 @@ const sendOrderConfirmedEmail = async (order, transaction) => {
   }
 };
 
+// Sends an email when a deposit address is shown to the user in the crypto contribution flow.
+// Here a pending order is created.
+const sendCryptoOrderProcessingEmail = async order => {
+  const { collective, fromCollective } = order;
+  const user = order.createdByUser;
+  const host = await collective.getHostCollective();
+
+  const data = {
+    order: order.info,
+    collective: collective.info,
+    host: host.info,
+    fromCollective: fromCollective.activity,
+    isCrypto: true,
```

Comment:
I don't think that's needed.

Reply:
Yep, my mistake, I forgot to remove this after refactoring. Done. 👍🏽